improver_tests/acceptance/SHA256SUMS

```diff
@@ -284,8 +284,8 @@ f1c40e18a0658c15d809a3299a6b61c0a1b48e432c8febb727033b3018bdc136  ./nowcast-feat
 3248408d72810ac43d1c79f1befca5f977406c2b5e665d3d2d46db65ea496855  ./nowcast-feature-branch/optical-flow-from-winds/20190101T0700Z-PT0000H00M-orographic_enhancement.nc
 20f8b3c7677cf2cae616077ce016a4699c66e80ef51fed68d6687f9af90110c8  ./nowcast-feature-branch/optical-flow-from-winds/20190101T0700Z-PT0000H00M-wind_components_on_pressure_levels.nc
 78748096d963acd768075855a6b31ca4288bda671798b2d073dafaccf6d3ed7a  ./nowcast-feature-branch/optical-flow-from-winds/20190101T0700Z-u1096_ng_radar_precip_ratecomposite_2km.nc
-5857cf75270ebd834d81ac5107097fe39e6d5fa78725aa0e460a0e5ca5ed3d14  ./nowcast-feature-branch/optical-flow-from-winds/kgo_15min.nc
-f5a1c1d8c024f04623e42d5c91164dcee8dc08a41a8a2e972be23f0932ac401b  ./nowcast-feature-branch/optical-flow-from-winds/kgo_30min.nc
+50b3aed5d3647f5c72f3f33a7ed23d817bd0395e6db92af14e5ae53a5d6e30b4  ./nowcast-feature-branch/optical-flow-from-winds/kgo_15min.nc
+b2fb4ef29e945a62ecd8540412ec1692002c7cb48475d329bfb38ba0748d84cb  ./nowcast-feature-branch/optical-flow-from-winds/kgo_30min.nc
```

Comment:
Just as a note, it looks like updating these SHAs, which aren't related to this PR, won't cause an issue, as this was already done in #1318.